### DISCUSSION TITLE
linux

### DISCUSSION THREAD

**Discussion by kcmccombs** [seanap/Plex-Audiobook-Guide#76] [Q&A]
has anyone set this up for linux thanks

**Comment by seanap** [comment]
The linux only setup that I use is the [Auto-M4b](https://github.com/seanap/auto-m4b) docker to convert everything to m4b and get everything into the right folder structure to then use the [beets.io audible plug-in](https://github.com/seanap/beets-audible) to tag and organize.
  
Pros:

- Linux only
- Auto-m4b is automated
- beets.io is super quick and easy to manage from CLI
- docker compose install

Cons:

- Beets.io plugin is Audible only.  If your book isn't in audible it won't tag it, and there is no manual override.  For non-audible books I still use Windows/mp3tag.
- Beets.io is CLI only
- Beets can't set the chapter titles on chapterized m4b's
